packages/next-adapter/package.json

```diff
@@ -51,16 +51,17 @@
     "rimraf": "^3.0.2"
   },
   "peerDependencies": {
-    "next": "^9",
-    "react": "^16",
-    "react-native-web": "^0.13.12"
+    "next": "^11",
```

Comment:
NextJS 11 is required, as this assumes that Webpack 5 is the default. <10.x used to have this behind a `future` flag

Reply:
👍